src/components/Signin/SigninForm.js

```diff
@@ -137,7 +146,7 @@ const SigninForm = () => {
 
         <div className="text-center">
           <p className="text-grey-dark text-sm">
-            Don't have an account?{' '}
+            {`Don't have an account?`}
             <a href="/signup" className="no-underline text-blue font-bold">
```

Comment:
Just out of curiosity, I've noticed sometimes you use <a> and sometimes <Link> is there any rule for these tags?

Reply:
I think the gatsby Link tag should be used for internal components while the other tag should be used for external links